.github/workflows/pr-link-check.yml

```
@@ -0,0 +1,63 @@
+name: Check Links In Pull Requests
+permissions: {}
+
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize, ready_for_review]
```

I think we should add `edited` here, so people can retrigger the check.

```suggestion
    types: [opened, edited, reopened, synchronize, ready_for_review]
```